### ISSUE TITLE
Do mobs remember your conversation?

### ISSUE THREAD

**Issue by tinker1234** [zizmax/CraftGPT#12]
Do mobs remember your conversation?

**Comment by zizmax** [comment]
Yes, there is a customizable retention/memory window. Just be aware that
the longer the window, the more expensive the API calls.
